apps/kyc-server/src/middleware/validation.ts

```
@@ -0,0 +1,42 @@
+import { Request, Response, NextFunction } from 'express';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)